inc/computervirtualmachine.class.php

```diff
@@ -260,8 +260,11 @@ static function showForVirtualMachine(Computer $comp) {
       echo "<div class='center'>";
 
       if (isset($comp->fields['uuid']) && ($comp->fields['uuid'] != '')) {
-         $where = "LOWER(`uuid`)".self::getUUIDRestrictRequest($comp->fields['uuid']);
-         $hosts = getAllDatasFromTable('glpi_computervirtualmachines', $where);
+         $hosts = getAllDatasFromTable(
+            self::getTable(), [
+               'uuid'   => self::getUUIDRestrictRequest($comp->fields['uuid'])
```

Comment:
Does request still works without usage of `LOWER` ?

Reply:
Hum... I don't know at all why there is a lower on all that :/